src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -263,29 +268,46 @@ def acr_queue(cmd,
         result = LongRunningOperation(cmd.cli_ctx)(client_registries.queue_build(
             build_request=build_request, resource_group_name=resource_group_name, registry_name=registry_name))
 
-        print("Queued a build with build-id: {}.".format(result.build_id))
+        size = os.path.getsize(tar_file_path)
```

Comment:
This seems to be the only place besides `acr_build_show_logs` that can raise, can you handle error specifically here and not have all code in a huge try block?